lib/beaker/host/cisco.rb

```diff
@@ -72,7 +72,7 @@ def prepend_commands(command = '', user_pc = '', opts = {})
       return user_pc unless command.index('vsh').nil?
 
       prepend_cmds = 'source /etc/profile;'
-      if self[:platform] =~ /cisco-5/
+      if self[:platform] =~ /cisco-nexus/
```

Comment:
@glennmatthews We have a IOS XR VM that we are using for testing.  There does not appear to be a need to use the 'ip netns exec' to get things that require networking to work.  There is also no 'sudo' installed.  Is this normally available on IOS XR systems?  The only thing we've need to do for our own Puppet test is make sure puppet binaries are in the PATH.